### PR TITLE
Minor and patch updates for communication react 

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What
Ran the command `rush upgrade-interactive`, selected the communication-react package, and selected all available patch updates.

# Why
OCE task of updating UI library https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/47006/Updating-Dependencies?anchor=review-rush-available-updates

# How Tested
Ran `rush update:full && rush build:all-flavors` and testing if CI will pass here

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by mgamis-msft** [comment]
The camera icon was updated in the new fluent icon package causing the snapshot updates

**Review by prabhjot-msft — COMMENTED** [review]
There is major snapshot update without any visible changes, am I missing anything or are these snapshots generated by mistake

**Review by prabhjot-msft — APPROVED** [review]
Just saw the last comment about camera icon, looks good!

File: packages/fake-backends/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
   "license": "MIT",
   "dependencies": {
     "@azure/communication-chat": "1.5.0-beta.1 || ^1.4.0",
-    "@azure/communication-common": "^2.3.0",
+    "@azure/communication-common": "^2.3.1",
```
Comment (dmceachernmsft) [inline packages/fake-backends/package.json:35]: 👀

**Review by dmceachernmsft — APPROVED** [review]

**Review by emlynmac — APPROVED** [review]

**Pull request merged**